README.md

```diff
@@ -549,6 +549,111 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Profiling
+Profiling can be used to monitor the impact of Secretless on CPU and Memory consumption. Currently, Secretless supports two types- CPU and Memory.
+
+**Prerequisites:**
+- **Homebrew**
+- Graphviz to visualize profiling results: ```$ brew install graphviz```
+
+**Configuration:**
+
+In order to configure individual profiles properly and receive desired results, perform the following:
+1. Build Secretless locally: ``` $ ./bin/build_darwin```
+
+2. Run a Postgres container named `pg-test` and wait for it to become ready:
+```
+docker run -d --name pg-test  postgres:9.3;
+
+# wait for pg-test
+docker exec pg-test bash -c "
+ while ! pg_isready -U postgres > /dev/null 2>&1;
+   do
+     >&2 echo "."
+     sleep 1
+ done
+ echo 'Postgres is up and ready'
+"
+```
+
+3. Create and populate a database named `test`:
+```
+docker exec pg-test psql -U postgres  -c "
+   CREATE USER test PASSWORD 'test';
+
+  CREATE SCHEMA test;
+  CREATE TABLE test.test ( id INTEGER PRIMARY KEY );
+  INSERT INTO test.test VALUES ( generate_series(0, 99999) );
+
+  GRANT ALL ON SCHEMA test TO test;
+  GRANT ALL ON test.test TO test;
+"
+```
+
+4. Check that the Postgres database is running:
```

Comment:
Maybe we should take this out ? we know the database is doing fine from the previous steps.